wandb/sdk/internal/internal_api.py

```diff
@@ -1076,25 +1136,19 @@ def push_to_run_queue(
 
                 if res is None or res.get("queueID") is None:
                     wandb.termerror(
-                        "Unable to create default queue for {}/{}. Run could not be added to a queue".format(
-                            entity, project
-                        )
+                        f"Unable to create default queue for {entity}/{project}. Run could not be added to a queue"
```

Comment:
another nit: mind changing the working here to look like the one on line 1066. I think it makes it a bit clearer to our users